pyproject.toml

```diff
@@ -211,7 +211,8 @@ select = [
     "T201", # print
     "C901", # complex-structure
     "TCH", # flake8-type-checking
-    "D" # pydocstyle
+    "D", # pydocstyle
+    "I" # isort
```

Comment:
I've been wanting some import sorting for a while, call it a pet peeve

Reply:
Yeah it's been killing me too.